setup/wix/EFToolsWillowMsi.wixproj

```diff
@@ -18,8 +17,14 @@
     <MsiRuntimeInputsPackagesDir>$(OutDir)\MsiRuntimeInputs\packages</MsiRuntimeInputsPackagesDir>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(SingletonMsiName)' == '' ">
-    <SingletonMsiName>EF6Tools</SingletonMsiName>
+  <PropertyGroup>
+    <EmbedCab Condition="'$(EmbedCab)' == ''">no</EmbedCab>
+    <SingletonMsiName Condition="'$(SingletonMsiName)' == ''">EF6Tools</SingletonMsiName>
```

Comment:
Is there actually somewhere where we pass this in or is this just you trying to stay close to the previous behavior?

Reply:
Just keeping the original behavior